src/prefect/cli/deployment.py

```diff
@@ -426,20 +464,39 @@ async def run(
         )
 
         flow_run = await client.create_flow_run_from_deployment(
-            deployment.id, parameters=parameters
+            deployment.id,
+            parameters=parameters,
+            state=Scheduled(scheduled_time=scheduled_start_time),
         )
 
     if PREFECT_UI_URL:
         run_url = f"{PREFECT_UI_URL.value()}/flow-runs/flow-run/{flow_run.id}"
     else:
         run_url = "<no dashboard available>"
 
+    if start_in:
+        scheduled_display = (
+            scheduled_start_time.in_tz(
+                pendulum.tz.local_timezone()
+            ).to_datetime_string()
+            + " ("
+            + pendulum.format_diff(scheduled_start_time.diff(now))
+            + ")"
+        )
+    elif start_at:
+        scheduled_display = scheduled_start_time.in_tz(
+            pendulum.tz.local_timezone()
+        ).to_datetime_string()
+    else:
+        scheduled_display = "now"
+
     app.console.print(f"Created flow run {flow_run.name!r}.")
     app.console.print(
         textwrap.dedent(
             f"""
         └── UUID: {flow_run.id}
         └── Parameters: {flow_run.parameters}
+        └── Scheduled start time: {scheduled_display}
```

Comment:
I see that this will now always have a `Scheduled start time`, would it also make sense for one of the `--start-in` or `--start-at` to have a default of `now` rather than `None` or no?

Reply:
Hm that might make the `--help` message clearer but then we won't be able to tell if they've passed both `--start-in` and `--start-at` or if one of them is just the default value. We could set `start_at` to "now" after we do that  check to simplify some of our code, but then we'll display a timestamp for the scheduled time instead of the literal string "now" which I thought might be a nicer UX? Maybe it's not though :)

Reply:
Regarding your first point, ah yes, that's true! As for the second point, yes we could set `start_at` after we've checked that it wasn't set. I think it might be nice to be able to compare the timestamp in the UI or if for whatever reason you were for running it programmatically then perhaps you would have a nicer record of the time it was created.

Reply:
Maybe we can have the best of both worlds and do "Scheduled start time: {TIMESTAMP} (now)" when it's scheduled for now?

Reply:
Hehe I like that!

Reply:
If they pass in `--start-in` and `--start-at`, `--start-at` is ignored because it's 2nd in the if/else statement.
```python
if start_in:
    do_stuff()
elif start_at:
    do_other_stuff()
```
 So technically a `start-at` default of "now" could be possible? Or maybe I overlooked something 🤔 But regardless, it might be nicer to try to handle it the way we already described above.

Reply:
Good point — we should detect this and exit with a nice message :)